### ISSUE TITLE
Tests at t/008_unicode_matching.t are failing

### ISSUE THREAD

**Issue by blabos-zz** [LoonyPandora/Dancer-Plugin-Passphrase#5]
```
t/008_unicode_matching.t ............ 1/8 
#   Failed test 'SHA-1 needs encoded text'
#   at t/008_unicode_matching.t line 20.
#                   ''
#     doesn't match '(?^i:Wide character in subroutine entry)'

#   Failed test 'SHA-256 needs encoded text'
#   at t/008_unicode_matching.t line 28.
#                   ''
#     doesn't match '(?^i:Wide character in subroutine entry)'
# Looks like you failed 2 tests of 8.
t/008_unicode_matching.t ............ Dubious, test returned 2 (wstat 512, 0x200)
```

**Comment by LoonyPandora** [comment]
Thanks for the report. There are a couple of workarounds for you.
1. Install using force. This is checking for an error condition caused by other modules bugs - it doesn't indicate a problem with this module at all.
2. (The "proper" solution) - Upgrade your version of `Digest::SHA`. You are using an outdated version, and should be using at least version 5.81. Older version of `Digest::SHA` have bugs

I'll update the test to make sure that it tests correctly depending on the version of `Digest::SHA`. I do specify a minimum version of `Digest::SHA` in the `META.json` file - but obviously that doesn't enforce an upgraded version to be installed.

tl;dr Upgrade `Digest::SHA`. If you can't do that, install `Dancer::Plugin::Passphrase` by force - it will work just fine. I'll fix the test to not annoy people.

**Comment by blabos-zz** [comment]
I'm using bcrypt so is secure to install with "force" anyway.

When I updated Digest::SHA, all tests passed successful so no workarounds are needed.

Thank you also for the tip with the version of Digest::SHA. I will put this restriction in my projects too.

Thank you :)
